resource_customizations/argoproj.io/Rollout/actions/actionDiscovery.lua

```diff
@@ -0,0 +1,11 @@
+actions = {}
```

Comment:
Looks like it is not very convenient to construct dynamic arrays in Lua. Each user will have to deal with it. I think it would be better to return map with action name as a key instead of array:

```
actions = {}

if obj.spec.paused ~= nil and obj.spec.paused then
    actions["resume"]={}
end

return actions
```